tests/test_suites/ShortSystemTests/test_limit_glibc_malloc_arenas.sh

```diff
@@ -0,0 +1,88 @@
+timeout_set "2 minutes"
+
+CHUNKSERVERS=3 \
+	MASTER_CUSTOM_GOALS="1 ec21: \$ec(2,1)" \
+	AUTO_SHADOW_MASTER="NO" \
+	MOUNT_EXTRA_CONFIG="sfscachemode=NEVER" \
+	CHUNKSERVER_EXTRA_CONFIG="MAGIC_DEBUG_LOG = $TEMP_DIR/log|LOG_FLUSH_ON=DEBUG" \
+	MASTER_EXTRA_CONFIG="MAGIC_DEBUG_LOG = $TEMP_DIR/log|LOG_FLUSH_ON=DEBUG" \
+	setup_local_empty_saunafs info
+
+cd "${info[mount0]}"
+
+# Generates some files in parallel
+function generateAndValidateFiles() {
+	echo "Generating the files"
+
+	for file in $(seq 1 100); do
+		size=$((file * 1024))
+		FILE_SIZE=${size} assert_success file-generate "file.${file}" &
+	done
+
+	wait
+}
+
+function getActualArenaLimitFromLog() {
+	grep -ioP '(?<=Setting glibc malloc arena max to )[0-9]+' "${TEMP_DIR}/log" | tail -n 1
+}
+
+function getVirtualMemoryForPid() {
+	pid=${1}
+	ps -o vsz= -p ${pid}
+}
+
+# Generate some files to populate the filesystem
+generateAndValidateFiles
+
+# Master is consuming more memory after restart, so the comparison is not fair.
+# This restart makes the later comparison in better conditions.
+saunafs_master_daemon restart
+saunafs_wait_for_all_ready_chunkservers
+
+arenas=2
+
+### Test the limit in the first chunkserver ###
+
+# Get initial values
+cs0Pid=$(saunafs_chunkserver_daemon 0 test 2>&1 | sed 's/.*pid: //')
```

Comment:
Consider adding `tr -d '\0'` to suppress the warning from `sed` ignoring null byte in input. 
```bash
cs0Pid=$(saunafs_chunkserver_daemon 0 test 2>&1 | tr -d '\0' | sed 's/.*pid: //')
```

Reply:
Done.